kernel/src/main/java/org/kframework/krun/KRunOptions.java

```diff
@@ -217,36 +216,30 @@ public SearchType searchType() {
     @Parameter(names="--depth", description="The maximum number of computational steps to execute or search the definition for.")
     public Integer depth;
 
-    @ParametersDelegate
-    public Experimental experimental = new Experimental();
-
-    public final class Experimental {
-
-        @Parameter(names="--statistics", description="Print rewrite engine statistics.", arity=1,
-                converter=OnOffConverter.class)
-        public boolean statistics = false;
+    @Parameter(names="--statistics", description="Print rewrite engine statistics.", arity=1,
+            converter=OnOffConverter.class)
+    public boolean statistics = false;
 
-        @Parameter(names="--debugger", description="Run an execution in debug mode.")
-        public boolean debugger = false;
+    @Parameter(names="--debugger", description="Run an execution in debug mode.")
+    public boolean debugger = false;
 
-        @Parameter(names="--ltlmc", description="Specify the formula for model checking at the commandline.")
-        public String ltlmc;
+    @Parameter(names="--ltlmc", description="Specify the formula for model checking at the commandline.")
+    public String ltlmc;
 
-        @Parameter(names="--ltlmc-file", description="Specify the formula for model checking through a file.")
-        public String ltlmcFile;
+    @Parameter(names="--ltlmc-file", description="Specify the formula for model checking through a file.")
+    public String ltlmcFile;
 
-        public boolean ltlmc() {
-            return ltlmc != null || ltlmcFile != null;
-        }
+    public boolean ltlmc() {
+        return ltlmc != null || ltlmcFile != null;
+    }
 
-        @ParametersDelegate
-        public SMTOptions smt = new SMTOptions();
+    @ParametersDelegate
+    public SMTOptions smt = new SMTOptions();
 
-        @Parameter(names="--native-libraries", description="Flags to pass to linker. Useful in defining rewriter plugins.",
-                listConverter=StringListConverter.class)
-        public List<String> nativeLibraries = Collections.emptyList();
+    @Parameter(names="--native-libraries", description="Flags to pass to linker. Useful in defining rewriter plugins.",
```

Comment:
This should probably be marked as deprecated because I think it only has meaning on the Ocaml backend.

Reply:
I'm also starting to think that going through and marking which backends respect each given option should be done as a separate PR. We can even make the tool check that only the correct options are used for a given backend, if we like (I think we do this for `--non-strict`).

Reply:
See #1937

Reply:
I've marked this as deprecated/ocaml backend only.